config/services.yml

```diff
@@ -57,7 +61,7 @@ services:
       $cache: '@prestashop.module.link_block.cache'
       $moduleRepository: '@prestashop.core.admin.module.repository'
       $languages: '@=service("prestashop.adapter.legacy.context").getLanguages(true, service("prestashop.adapter.shop.context").getContextShopID())'
-      $shopId: '@=service("prestashop.adapter.shop.context").getContextShopID()'
+      $shopContext: '@prestashop.adapter.shop.context'
```

Comment:
It was much easier to base on `shopContext` instead of having more parameters

if that's too much of a BC break I'll move it to separated ones

Reply:
BC breaks are allowed in this PR, let's not multiply variables in services.yml 👍